### PR TITLE
Master

### PR DESCRIPTION
**Pull request by Hooooolly**
Add removeItem intent

Three tests in removeItemTest:
1. remove item that is there
2. remove item that is not there
3. invalid intput

Update index and SampleUtterances for RemoveItem part

### CODE REVIEW

**Review by cwlucas41 — APPROVED** [review]
good work

File: speechAssets/SampleUtterances.txt [hunk 1]
```diff
@@ -3,3 +3,9 @@ AddItem I have a {Item}
 AddItem I have an {Item}
 AddItem I bought a {Item}
 AddItem I bought an {Item}
+
+RemoveItem Remove {Item}
+RemoveItem I’m out of {Item}
+RemoveItem I am out of {Item}
+RemoveItem I ran out of {Item}
+RemoveItem There is no {Item} lefts
```
Comment (bkelly0729) [inline speechAssets/SampleUtterances.txt]: Change "There is no {Items} lefts" to "There is no {Items} left".

**Review by bkelly0729 — COMMENTED** [review]
Looks good!

**Comment by TEWatson** [comment]
🌏

**Pull request merged**